Python/Covid_Cases_Scraper/README.md

```diff
@@ -0,0 +1,59 @@
+# CoVID Cases Scraper
+
+For CoVID-19 Cases we are using the concept of web scraping, where we are fetching the data from a website called Worldometers. In this project we can fetch the data of Coronavirus (CoVID-19) cases for any country in the World. 
+
+To do this we need two libraries `beautifulsoup4` and `requests`
+
+### Beautifulsoup4
+
+Beautiful Soup is a library that makes it easy to scrape information from web pages. It sits atop an HTML or XML parser, providing Pythonic idioms for iterating, searching, and modifying the parse tree.
+
+### Requests 
+
+Requests allows you to send HTTP/1.1 requests extremely easily. There’s no need to manually add query strings to your URLs, or to form-encode your PUT & POST data — but nowadays, just use the json method!
+
+## Installation
+
+Install with `pip` command in any terminal
```

Comment:
Provide instructions to use `requirements.txt` as well

Reply:
done